docs/evm/clients/web3-js.md

```diff
@@ -24,15 +24,15 @@ npm install web3
 
 ## Initializing Web3 with Flow
 
-To use `web3` in your project, start by importing the module and initializing your `Web3` instance with a Flow RPC endpoint.  This example uses [Flow Previewnet](https://developers.flow.com/networks/flow-networks/accessing-previewnet).
+To use `web3` in your project, start by importing the module and initializing your `Web3` instance with a Flow RPC endpoint. This example uses [Flow Previewnet](https://developers.flow.com/networks/flow-networks/accessing-previewnet).
 
 ```js
-const { Web3 } = require('web3')
-const web3 = new Web3('https://previewnet.evm.nodes.onflow.org')
+const { Web3 } = require('web3');
+const web3 = new Web3('https://previewnet.evm.nodes.onflow.org');
```

Comment:
I like how you use mainnet endpoint in code and just note about using testnet. Previewnet likely not gonna be around for much longer, so we'll need to go back in and remove soon anyway

Reply:
Updated web3.js to remove all references to previewnet and use same flow as ethers.js.